roles/perf/tasks/main.yml

```diff
@@ -0,0 +1,93 @@
+---
+# Run performance number gathering tasks tasks
+
+# Fact collection and initialization block
+- block:
+     # Check for mandatory variables, fail if not set
+     - name: Check if mandatory variables are set
+       assert:
+         that:
+            - "{{item}} is defined"
+         fail_msg: "FAIL: variable {{item}} not set"
+         success_msg: "PASS: variable {{item}} is defined."
+       run_once: true
+       delegate_to: localhost
+       with_items:
+         - gluster_maintenance_volname
+         - gluster_maintenance_mountpoint
+         - gluster_maintenance_client
```

Comment:
what's the difference between gluster_maintenance_client and gluster_maintenance_vm ?

Reply:
gluster_maintenance client is the GlusterFS client. And vm is the virtual machine stored on GlusterFS.